### PR TITLE
doc: update restore doc

### LINKED ISSUE #755: Test whether the Rancher backup is functional

### PR DESCRIPTION
**Pull request by maxyzli**
Thanks for @vthwang helps.

resolves #755

### CODE REVIEW

File: docs/rancher/03-setup-rancher/disaster-recovery-rancher.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+- [Step 2 - Install the rancher-backup Helm chart](#step-2---install-the-rancher-backup-helm-chart)
+- [Step 3 - Restore from backup using a Restore custom resource](#step-3---restore-from-backup-using-a-restore-custom-resource)
+- [Step 4 - Follow the Setting Up Rancher on k3s Guide](#step-4---follow-the-setting-up-rancher-on-k3s-guide)
+- [Conclusion](#conclusion)
+
+## Prerequisites
+
+Before beginning the restoration process, ensure you have:
+
+- A newly set up server following the [Setup Ubuntu](../01-setup-ubuntu/README.md) guide.
```
Comment (vthwang) [inline docs/rancher/03-setup-rancher/disaster-recovery-rancher.md:21]: We also need to setup k3s before restoring the backup.
https://github.com/MurmurationsNetwork/MurmurationsServices/blob/main/docs/rancher/02-setup-k3s/README.md

File: docs/rancher/03-setup-rancher/disaster-recovery-rancher.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+### 1. Add the Helm repository
+
+```bash
+helm repo add rancher-charts https://charts.rancher.io
+helm repo update
+```
+
+### 2. Set a `CHART_VERSION` variable
+
+Refer to the [support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-8-3/) under the Rancher Apps / Cluster Tools section to determine which rancher-backup versions are compatible. For instance, if you are using Rancher version `v2.8.3`, you should set the chart version to `103.0.1+up4.0.1`.
```
Comment (vthwang) [inline docs/rancher/03-setup-rancher/disaster-recovery-rancher.md:43]: Use this link can directly jump to Rancher Apps / Cluster Tools section
https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-8-3/#el-59173

File: docs/rancher/03-setup-rancher/disaster-recovery-rancher.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+- [Step 3 - Restore from backup using a Restore custom resource](#step-3---restore-from-backup-using-a-restore-custom-resource)
+- [Step 4 - Follow the Setting Up Rancher on k3s Guide](#step-4---follow-the-setting-up-rancher-on-k3s-guide)
+- [Conclusion](#conclusion)
+
+## Prerequisites
+
+Before beginning the restoration process, ensure you have:
+
+- A newly set up server following the [Setup Ubuntu](../01-setup-ubuntu/README.md) guide.
+- A backup file ready for restoration. Check the DigitalOcean space.
```
Comment (vthwang) [inline docs/rancher/03-setup-rancher/disaster-recovery-rancher.md:22]: Also, I think it's worth to note that we need to setup DNS record. We can emphasize it with bold text.

File: docs/rancher/README.md [hunk 4]
```diff
@@ -9,6 +9,7 @@ This guide offers developers a comprehensive walkthrough on initiating an operat
     - [Upgrade K3s](02-setup-k3s/upgrade-k3s.md)
 3. [Set Up Rancher](03-setup-rancher/README.md)
     - [Backup Rancher](./03-setup-rancher/backup-rancher.md)
```
Comment (vthwang) [inline docs/rancher/README.md:11]: Although it does not affect reading, I feel it's better to unify the paths and remove all the leading `./`.

**Review by vthwang — APPROVED** [review]
LGTM 🚀

**Review by geoffturk — APPROVED** [review]
LGTM 🚀

**Pull request merged**